### ISSUE TITLE
Set AV Sync Adjustment

### ISSUE THREAD

**Issue by jpleite** [bendavid/aiopylgtv#52]
Hi. A quick question: I checked the endpoints and it seems there's no way to change the AV Sync Adjustment. Do you confirm?

**Comment by bendavid** [comment]
It's possible to do almost anything with luna_request

**Comment by jpleite** [comment]
Right. But I checked https://webostv.developer.lge.com/api/webos-service-api/intro-lunaservice/ and did not find any endpoint to change the AV Sync Adjustment unless I'm checking it the wrong way

**Comment by bendavid** [comment]
I suspect it's possible through com.webos.settingsservice/setSystemSettings though I don't know what the exact setting is.

**Comment by jpleite** [comment]
Any idea how can I find a list of available commands using getSystemSettings?

**Comment by chros73** [comment]
Which model do you have? There're avSync* properties at least with CX/C1, see them [here](https://github.com/chros73/bscpylgtv/tree/master/docs).

**Comment by jpleite** [comment]
Thanks for the tip. I have a OLED65CX5LB so it should work. I just don't know how to set the avSync via Home Assistant service: webostv.command. Any guidelines would be much appreciated
